tests/e2e/network/service_annotations.go

```diff
@@ -448,10 +448,8 @@ var _ = Describe("Service with annotation", FlakeAttempts(3), Label(utils.TestSu
 	})
 
 	It("should support service annotation `service.beta.kubernetes.io/azure-pip-prefix-id`", func() {
-		if skuEnv := os.Getenv(utils.LoadBalancerSkuEnv); skuEnv != "" {
-			if !strings.EqualFold(skuEnv, string(network.PublicIPAddressSkuNameStandard)) {
-				Skip("pip-prefix-id only work with Standard Load Balancer")
-			}
+		if !strings.EqualFold(os.Getenv(utils.LoadBalancerSkuEnv), "standard") {
```

Comment:
network.PublicIPAddressSkuNameStandard
```suggestion
		if !strings.EqualFold(os.Getenv(utils.LoadBalancerSkuEnv), string(network.PublicIPAddressSkuNameStandard) {
```

Reply:
Updated. Some similar codes are also modified.